Packs/PaloAltoNetworks_IoT/Playbooks/playbook-iot-playbook.yml

```diff
@@ -0,0 +1,280 @@
+id: PANW IoT Incident Handling
```

Comment:
Why not merge this playbook with the serviceow one (PANW IoT Incident Handling with ServiceNow)? you can add a playbook input that checks if to open a ticket or not
besides that the playbooks look the same so it seems redundant

Reply:
If I have a step like this to merge two playbooks,

what if I have the ServiceNow integration enabled for other things, but I do "not" want to create a record in this playbook?

Reply:
What I suggested here is an input to check if the user wants to open a ticket, not check if the integration is enabled

For example the input name will be
CreateServiceNowTicket
Value can be true or false
You have a conditional task in the playbook to check if the user specified in the inputs true or false
If true your task will be create ticket, if false go to done etc

Reply:
added the input